build/config/android/config.gni

```diff
@@ -94,20 +94,19 @@ if (is_android) {
 
   # Subdirectories inside android_ndk_root that contain the sysroot for the
   # associated platform.
-  _android_api_level = 16
+  _android_api_level = 24
```

Comment:
Generally, the Android NDK platform level is orthogonal to the Android API level. This line just means that the toolchain assumes headers and libraries from Android Nougat are present. We still need to do the due diligence to not use features that are absent on older versions on Android (like Vulkan, which we try to `dlopen` and fall back to OpenGL ES). Likewise, these builds will still work on newer versions of Android. So really, this does not enforce a maximum or minimum version compatibility.